src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -624,7 +627,11 @@ export function getExpressionEnrichmentColumns(
             name: group.name,
             headerRender: (name: string) => STAT_IN_headerRender('μ', name),
             render: (d: ExpressionEnrichmentRow) => (
-                <span>{d.groupsSet[group.name].meanExpression.toFixed(2)}</span>
+                <span>
+                    {d.groupsSet[group.name]
+                        ? d.groupsSet[group.name].meanExpression.toFixed(2)
+                        : 'NA'}
```

Comment:
lets put this into a constant for reuse. also in general I think we are using N/A so maybe we should stick with that?